tests/wicked/advanced/sut/t15_macvtap_legacy.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Advanced test cases for wicked
+# Test 15: Create a macvtap interface from legacy ifcfg files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use warnings;
+use testapi;
+use network_utils 'iface';
+use utils 'zypper_call';
+
+our $macvtap_log = '/tmp/macvtap_results.txt';
+
+sub run {
+    my ($self) = @_;
+    record_info('Info', 'Create a macvtap interface from legacy ifcfg files');
+    my $config     = '/etc/sysconfig/network/ifcfg-macvtap1';
+    my $iface      = iface();
+    my $ip_address = $self->get_ip(type => 'macvtap', netmask => 1);
+    $ip_address =~ s'/'\\/';
+    my $ref_ip = $self->get_ip(type => 'host', netmask => 0, is_wicked_ref => 1);
+    $self->get_from_data('wicked/ifcfg/macvtap1',    $config);
+    $self->get_from_data('wicked/ifcfg/macvtap_eth', '/etc/sysconfig/network/ifcfg-' . $iface);
+    $self->get_from_data('wicked/check_macvtap.c',   'check_macvtap.c', executable => 1);
+    zypper_call('in gcc');
+    assert_script_run('gcc ./check_macvtap.c -o check_macvtap');
+    script_run('chmod +x ./check_macvtap');
+    assert_script_run("sed 's/iface/$iface/' -i $config");
```

Comment:
`./` not needed in these two (of course not a problem).

Reply:
I prefer to have it